Makefile

```diff
@@ -1,6 +1,7 @@
 export GO111MODULE=on
 TOP_LEVEL=$(shell git rev-parse --show-toplevel)
 COMMIT_HASH=$(shell git describe --always --tags --long)
+COMMIT_LONGHASH=$(shell git rev-parse HEAD)
```

Comment:
Let's use the latest tag, ideally `git describe --tags --abbrev=0`.
But make sure this works for release builds. You may need to overwrite this in the github action using one of the environment variables: https://docs.github.com/en/actions/learn-github-actions/environment-variables